tweets/2020/npm-automation-tokens.tweet

```diff
@@ -0,0 +1,5 @@
+📯 semantic-release release now works with the new npm automation tokens. That means you can enable two-factor auth on your account, while still automate releases through CI.
```

Comment:
```suggestion
📯 semantic-release release now works with the new npm automation tokens. That means you can enable two-factor auth on your account, while still automating releases through CI.
```

Reply:
Ha! I knew it was a good idea to request a review :D